pkg/license/license.go

```diff
@@ -10,13 +10,65 @@ import (
 	kotsv1beta1 "github.com/replicatedhq/kots/kotskinds/apis/kots/v1beta1"
 	"github.com/replicatedhq/kots/kotskinds/client/kotsclientset/scheme"
 	"github.com/replicatedhq/kots/pkg/buildversion"
+	"github.com/replicatedhq/kots/pkg/logger"
+	"github.com/replicatedhq/kots/pkg/store"
 )
 
 type LicenseData struct {
 	LicenseBytes []byte
 	License      *kotsv1beta1.License
 }
 
+func ResolveExistingLicense(verifiedLicense *kotsv1beta1.License) (bool, error) {
+	installedAppSet := make(map[string]bool)
+
+	apps, err := store.GetStore().ListInstalledApps()
+	if err != nil {
+		logger.Error(errors.Wrap(err, "failed to list installed apps"))
+		return false, err
+	}
+
+	notInstalledApps, err := store.GetStore().ListNotInstalledApps()
```

Comment:
let's call it "failed" instead of "not installed"?
```suggestion
	notInstalledApps, err := store.GetStore().ListFailedApps()
```